charts/prerequisites/Chart.yaml

```
@@ -21,12 +21,12 @@ dependencies:
     repository: https://equinor.github.io/helm-charts/charts/
     condition: neo4j-community.enabled
   - name: mysql
-    version: 9.1.2
+    version: 9.1.8
     repository: https://charts.bitnami.com/bitnami
```

```suggestion
    repository: https://raw.githubusercontent.com/bitnami/charts/archive-full-index/bitnami
```
at some point the will be removed from their index.html